docs/en/connector-v2/sink/Jdbc.md

```diff
@@ -33,6 +33,7 @@ support `Xa transactions`. You can set `is_exactly_once=true` to enable it.
 | user                                      | String  | No       | -             |
 | password                                  | String  | No       | -             |
 | query                                     | String  | No       | -             |
+| driver_type                               | String  | No       | -             |
```

Comment:
Hi, Thanks for you contribution. Can you add OceanBase Source and Sink Document like https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/Mysql.md and https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/sink/Mysql.md